vulcanai/models/metrics.py

```diff
@@ -62,6 +62,8 @@ def get_score(targets, predictions, metrics='accuracy', average=None,
         functions_without_average_parameter = ["get_accuracy", "get_mse"]
 
         if isinstance(targets, torch.Tensor):
+            # TODO: this is not necessary if targets was created as numpy on
+            # the CPU and then converted using from_numpy
```

Comment:
is there way to do a check for this?

Reply:
hm good point will look into

Reply:
As far as I can tell it appears not @rfratila, although this was interesting reading https://stackoverflow.com/questions/48482787/pytorch-memory-model-torch-from-numpy-vs-torch-tensor

Reply:
perhaps we should modify the condition to make it more useful then? such as checking if on a GPU device?

Reply:
So I think we want to use .detach regardless cause it adds in requires_grad=False, and then as far as I can tell .cpu is a no-op if it's already on the the CPU, and then we'll have to convert to numpy in any case.

Reply:
sounds good. the condition can then make it support both torch tensors and numpy matrices